src/nvme/types.h

```diff
@@ -308,6 +317,8 @@ enum nvme_cc {
 	NVME_CC_SHN_NONE	= 0,
 	NVME_CC_SHN_NORMAL	= 1,
 	NVME_CC_SHN_ABRUPT	= 2,
+	NVME_CC_CRWME       = 0,
+	NVME_CC_CRIME       = 1,
```

Comment:
Please align the = with the other ones right above.

Reply:
Fixed this